core/templates/pages/preferences-page/preferences-page.component.spec.ts

```diff
@@ -196,21 +201,22 @@ describe('Preferences Controller', function() {
     $httpBackend.verifyNoOutstandingRequest();
   });
 
-  it('should save default dashboard', function() {
-    var isRequestTheExpectOne = function(queryParams) {
-      return decodeURIComponent(queryParams).match(
-        '"update_type":"default_dashboard"');
-    };
-    $httpBackend.expect(
-      'PUT', '/preferenceshandler/data', isRequestTheExpectOne).respond(200);
-    ctrl.saveDefaultDashboard({});
-    $httpBackend.flush();
-
-    $httpBackend.verifyNoOutstandingExpectation();
-    $httpBackend.verifyNoOutstandingRequest();
-  });
+  it('should save default dashboard on backend when saving dashboard',
+    function() {
+      var isRequestTheExpectOne = function(queryParams) {
+        return decodeURIComponent(queryParams).match(
+          '"update_type":"default_dashboard"');
+      };
+      $httpBackend.expect(
+        'PUT', '/preferenceshandler/data', isRequestTheExpectOne).respond(200);
+      ctrl.saveDefaultDashboard({});
+      $httpBackend.flush();
+
+      $httpBackend.verifyNoOutstandingExpectation();
+      $httpBackend.verifyNoOutstandingRequest();
+    });
 
-  it('should handle export data click', function() {
+  it('should export data when handling export data click', function() {
```

Comment:
I don't understand this test. Can you explain where the handleExportDataClick() function is used?

Reply:
It's being used when exporting account.